docs/index.rst

```diff
@@ -6,54 +6,76 @@ provides a simple interface for performing aperture photometry of either a singl
 
-Creating an environment with `conda` or `mamba`:
+Only use one of the methods below for making a virtual environment.
 
-```bash
-mamba create -n stellarphot-test python=3.11
-mamba activate stellarphot-test
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `conda` or `mamba`::
 
-Creating an environment with `virtualenv`:
+    mamba create -n stellarphot-test python=3.11
+    mamba activate stellarphot-test
+    pip install git+https://github.com/mwcraig/stellarphot.git@update-docs
 
-```bash
-python3 -m venv stellarphot-test
-source stellarphot-test/bin/activate
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `virtualenv`::
+
+
+    python3 -m venv stellarphot-test
+    source stellarphot-test/bin/activate
+    pip install --pre --upgrade stellarphot
+
+
+Getting Started
+===============
 
 Overview
 --------
 
+You will go through this process to do photometry.
+
+#. You need to make some settings, like camera properties, observatory information, and passband maps. You may only need
+   to do this step once if you use the same equipment for all of your observations.
+#. Settings specific to an object need to be made:
+
+   #. night of data, like the photometry aperture radius, need to be made.
+   #. a list of the sources for which you want to perform photometry. These lists can be re-used.
+
+#. Review all of the settings that the photometry routines will use.
+#. Once those settings have been done, you can perform photometry on your images.
+
 
+Graphical interface for making settings
+---------------------------------------
 
-Graphical interface for generating settings
--------------------------------------------
+If you open up JupyterLab, the launcher should have a section that looks like this:
+
+.. image:: /_static/launcher.png
+    :width: 400px
+    :alt: JupyterLab Launcher with stellarphot notebooks
+
+Each of the notebooks corresponds to the steps in the previous section. Open each notebook in order, and run
+all of the cells in the notebook. In each will be a graphical interface to enter the camera and other settings
+(in notebook 1), measuring the seeing and choose comparison stars (in notebook 2), review all of your settings
+(in notebook 3), and perform photometry (in notebook 4).
+
+When the photometry is done there will be a new notebook called `photometry_run.ipynb` that will have a record
+of the photometry that was done.
 
-To generate settings using a graphical interface, start JupyterLab. In the launcher will be a section called
-"Stellarphot" with a link to "Generate Settings". Clicking on this link will open a notebook where you can enter settings.
 
 Command line interface for generating settings
 ----------------------------------------------
 
-To generate settings using the command line, run the following command:
+To generate settings using the command line, run the following command::
+
+    stellarphot-settings
```

Comment:
My attempt at this resulted in
```
zsh: command not found: stellarphot-settings
```

Reply:
I attempted in both the Jupyter terminal and my miniforge promt and got this error 

```
stellarphot-settings : The term 'stellarphot-settings' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ stellarphot-settings
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (stellarphot-settings:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Reply:
I opened #388 to track this. I don't think there is time to get this going (beyond maybe something very bare bones) for 2.0....

Reply:
...though PRs are welcome 😀